samples/highcharts/blog/nw-landing-page/demo.js

```diff
@@ -0,0 +1,1185 @@
+    const chart = this,
+        series = chart.series[0];
+    let newData = [];
+    let count = 0;
+    setInterval(function () {
+        const pname = planets[count];
+        newData = buildData(pname);
+        series.update({
+            data: newData
+        });
+        count = count + 1;
+        if (count === planets.length - 1) {
+            count = 0;
+        }
+    }, 3000);
+}
+);
+
+/*LINE GRAPH*/
+Highcharts.chart('graph', {
+    chart: {
+        margin: 0,
+        animation: {
+            duration: 1000,
+            easing: 'easeInQuint'
+        },
+        events: {
+            load: function () {
+                const chart = this;
+                let reversed = false;
+                setInterval(function () {
+                    if (reversed === false) {
+                        reversed = true;
+                    } else {
+                        reversed = false;
+                    }
+                    chart.yAxis[0].update({
+                        reversed: reversed
+                    });
+                }, 5000);
+            }
+        }
+    },
+    title: {
+        text: ''
+    },
+    exporting: {
+        enabled: false
+    },
+    credits: {
+        enabled: false
+    },
+    xAxis: {
+        gridLineWidth: 1,
+        min: -5,
+        max: 5,
+        tickInterval: 1,
+        lineColor: 'transparent',
+        offset: -150
+    },
+    yAxis: {
+        min: -5,
+        max: 5,
+        tickInterval: 1,
+        lineWidth: 1,
+        lineColor: 'transparent',
+        offset: -150,
+        title: {
+            text: null
+        }
+    },
+    legend: {
+        enabled: false
+    },
+    tooltip: {
+        enabled: false
+    },
+    series: [{
+        label: {
+            enabled: false
+        },
+        data: [
+            [-3, 3],
+            [-2, 2],
+            [-1, 1],
+            [0, 0],
+            [1, 1],
+            [2, 2],
+            [3, 3]
+        ]
+    }]
+});
+
+/*NETWOR GRAPH*/
+const dirDist50 = '#E8544E';
+const dirDist10 = '#FFD265';
+const dirDistLess10 = '#2AA775';
+Highcharts.chart('network', {
+    chart: {
+        type: 'networkgraph',
+        marginTop: 0,
+        animation: {
+            duration: 1000,
+            easing: 'easeInQuint'
+        },
+        events: {
+            load: function () {
+                const chart = this;
+                let mass = 10;
+                setInterval(function () {
+                    if (mass === 10) {
+                        mass = 1000;
+                    } else {
+                        mass = 10;
+                    }
+                    chart.series[0].nodes[0].update({
+                        mass: mass
+                    });
+                }, 5000);
+            }
+        }
+    },
+    exporting: {
+        enabled: false
+    },
+    credits: {
+        enabled: false
+    },
+    title: {
+        text: ''
+    },
+    tooltip: {
+        enabled: false,
```

Comment:
Why do we disable the tooltip (but still have advanced formatters for it)? If we can, I think we should keep the tooltip as it is a great interactive and assistive feature. The demos are nicely set up with real world examples like planets or airports, all of this is lost if we don't have the tooltip.

Reply:
I put the tooltips back and fixed the first graph so it actually means something.